contracts/staking/BaseShareField.sol

```diff
@@ -70,7 +71,7 @@ contract BaseShareField is DAOContract{
         if(block.number >= blockStart && lastRewardBlock < blockStart) lastRewardBlock = blockStart;
         if(block.number >= blockStart && blockEnd>= block.number){
             uint256 multiplier = block.number - lastRewardBlock;
-            uint256 reward = multiplier * (rewardsPerBlock * 1e16); // turn it to 18 decimals
+            uint256 reward = multiplier * (rewardsPerBlock * 1e16); // rewardsPerBlock is in G$ which is only 2 decimals, we turn it into 18 decimals
 
             accAmountPerShare = accAmountPerShare + (reward * 1e12 / totalProductivity); 
```

Comment:
why 1e12

Reply:
Acc amount share in 12 decimals can increase resolution if necessary? @sirpy